tests/index.spec.ts

```diff
@@ -0,0 +1,35 @@
+import type { ExecaChildProcess } from "execa";
+import { execa } from "execa";
+import * as path from "path";
+import { fileURLToPath } from 'url';
+
+const EXTENSIONS = ".ts,.tsx,.cts,.mts";
+const NO_IGNORE_FLAG = "--no-ignore";
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+const eslintModulePath = path.resolve(__dirname, "../node_modules/.bin/eslint");
+
+async function lintProject(projectName: string): Promise<ExecaChildProcess<string>> {
+  const projectPath = path.resolve(__dirname, "fixtures", projectName);
+  const commandArguments = ["--ext", EXTENSIONS, NO_IGNORE_FLAG];
+
+  try {
+    return await execa(
+      eslintModulePath,
+      [`${projectPath}/`, ...commandArguments],
+      {
+        cwd: projectPath,
+      }
+    );
+  } catch (error) {
+    return error;
```

Comment:
non-blocing: given what we [discussed here](https://github.com/qonto/eslint-config-typescript/pull/1/files#r1368589751), maybe it is better to log the error instead of returning it. In case of error the test will fail for sure, but we wouldn't have any knowledge about the problem.
What do you think?

```suggestion
    return error;
```

Reply:
Actually, we want to return the error because, in the context of the test, we expect the script run via `execa` to fail. Our project sample contains lint errors on purpose, so we want the call to `eslint --ext .ts,.tsx,.cts,.mts --no-ignore` to fail with `exitCode: 1` due to these lint errors.

Reply:
Gotcha! 👍 LGTM